sympy/solvers/solveset.py

```diff
@@ -978,6 +985,319 @@ def _solveset(f, symbol, domain, _check=False):
     return result
 
 
+def _expo_solver(f):
+    """
+    Helper function for solving exponential equations.
+
+    Exponential equations are the type of equations which includes a
+    variable located in the exponent.
+    For example `5**(2*x + 3) - 5**(3*x - 1)`.
+    Equations having two arguments (like the one above) or that can be
+    reduced to two argument form can be handled by this helper, other
+    variants require numerical methods to evaluate.
```

Comment:
There is no need to mention that, that's an obvious fact which we are not interested in.